build.gradle

```diff
@@ -60,17 +60,6 @@ compileJava {
             ".*/generated-sources/.*"
 }
 
-license {
```

Comment:
Whey are these removed?

Reply:
These two sections are currently duplicated...See above.
Probably we had a bad merge previously.

Reply:
I see.